<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeeProducts</class>
 <widget class="QMainWindow" name="SeeProducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="minp_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>min_price:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_le">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>31</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxp_lbl">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>31</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max_price:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_spinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>31</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="category_le">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>31</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_spinBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>31</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="value">
     <number>20000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="category_lbl">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>31</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="city_lbl">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>city:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>100</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Products</string>
    </property>
    <widget class="QTableWidget" name="products_table">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>31</y>
       <width>761</width>
       <height>351</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>City</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_btn">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>490</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Show product</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
